libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/instruction/InstructionView.java

```diff
@@ -176,18 +181,21 @@ public void onFeedbackDismissed() {
    * @param navigationViewModel to which this View is subscribing
    * @since 0.6.2
    */
-  public void subscribe(NavigationViewModel navigationViewModel) {
+  public void subscribe(LifecycleOwner owner, NavigationViewModel navigationViewModel) {
```

Comment:
Let's make sure we update the release notes and the android docs site with these changes (and add the `SEMVER` label here 😅), as we are breaking `subscribe` API here 👍

Reply:
Done